modules/installation/proc-installing-rhdh-on-kubernetes-with-helm-cli.adoc

```diff
@@ -0,0 +1,55 @@
+[id="installing-rhdh-on-kubernetes-with-helm-cli_{context}"]
+= Installing {product} on Kubernetes with Helm CLI
+
+You can use the Helm CLI to install {product} on the following platforms:
+
+* {eks-brand-name} ({eks-short})
+* {aks-brand-name} ({aks-short})
+
+.Prerequisites
+* A `kubectl` session, with `developer` or `admin` permissions, on a Kubernetes cluster.
+* You have installed the link:https://helm.sh/docs/intro/install[Helm CLI].
+
+.Procedure
+. Add the OpenShift Helm Charts repository.
```

Comment:
See https://github.com/openshift-helm-charts/charts?tab=readme-ov-file#openshift-helm-charts or https://charts.openshift.io/

Helm and Charts is capitalized.

Reply:
I was going by the official Helm docs: https://helm.sh/docs/chart_template_guide/getting_started/
It may be capitalized in the `openshift-helm-charts` repo, but it's not in the official product docs: https://docs.openshift.com/container-platform/4.15/applications/working_with_helm_charts/understanding-helm.html

Reply:
OpenShift here should probably also use the product attribute

Reply:
Seems we use both in the existing 1.1 docs:

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/227597/85b48c4e-e138-41f5-ba6f-2f4c7e079ae8)

Reply:
https://charts.openshift.io/ The repository name, OpenShift Helm Charts, is a proper noun and should be capitalized as such.